extensions/yaml_support/labgraph_monitor/server/serializer_node.py

```diff
@@ -24,8 +26,17 @@ class Serializer(lg.Node):
     TOPIC = lg.Topic(WSStreamMessage)
     config: SerializerConfig
 
+    # Test Input
+    INPUT = lg.Topic(RandomMessage)
+
+    def update(self, _in: SerializedGraph, data) -> SerializedGraph:
+        _in['nodes']['RollingAverager']['upstreams']['NoiseGenerator'][0]['fields']['data']['content'] = data
+        # print(_in)
+        return _in
```

Comment:
hey @bennaaym could please check it out? I can't seem to update the `SerializedGraph` dictionary to **include the message data** that is coming from the subscriber `INPUT` in this function `update`.